spec.html

```diff
@@ -25585,13 +25585,12 @@ <h1>Number.prototype.toPrecision ( _precision_ )</h1>
       <emu-clause id="sec-number.prototype.tostring">
         <h1>Number.prototype.toString ( [ _radix_ ] )</h1>
         <emu-note>
-          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is not present or is *undefined* the Number 10 is used as the value of _radix_.</p>
+          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is *undefined*, 0 is assumed.</p>
```

Comment:
```suggestion
          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is *undefined*, 10 is assumed.</p>
```

Reply:
Fixed in 9918c5b: "suggested change" feature seems broken.